iosApp/iosApp/Localizable.xcstrings

```diff
@@ -17513,8 +17560,8 @@
               },
               "zero" : {
                 "stringUnit" : {
-                  "state" : "new",
-                  "value" : "This stop has %ld elevators closed"
+                  "state" : "needs_review",
+                  "value" : "Estasyon sa a gen %ld asansè fèmen"
```

Comment:
Caught that this translation had been missing before; should we be rejecting anything with a state of `"new"` in CI?